data/193/046/663/3/1930466633.geojson

```diff
@@ -0,0 +1,82 @@
+{
+  "id": 1930466633,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"2022-01-01",
+    "geom:area":0.29362,
+    "geom:bbox":"-72.953849,41.544725,-72.156321,42.037528",
+    "geom:latitude":41.818511,
+    "geom:longitude":-72.57205,
+    "iso:country":"US",
+    "label:eng_x_preferred_longname":[
+        "Capitol Planning Region"
+    ],
+    "label:eng_x_preferred_placetype":[
+        "planning region"
+    ],
+    "lbl:latitude":41.81697,
+    "lbl:longitude":-72.575886,
+    "lbl:max_zoom":10.0,
+    "lbl:min_zoom":8.0,
+    "mz:hierarchy_label":0,
+    "mz:is_current":1,
+    "mz:min_zoom":6.0,
+    "name:eng_x_preferred":[
+        "Capitol"
+    ],
+    "src:geom":"uscensus",
+    "src:lbl":"uscensus",
```

Comment:
Nit: this should be `src:lbl_centroid`

The new records look great otherwise 👍

Reply:
Fixed in [cfcfe4e](https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/207/commits/cfcfe4edcec60eda0e042663784a751b5d537574)